### PR TITLE
chore(deps): update all non-major dependencies

### PR DESCRIPTION
**Pull request by czi-frontend-infra**
This PR contains the following updates:

| Package | Type | Change |
|---|---|---|
| [@chanzuckerberg/axe-storybook-testing](https://togithub.com/chanzuckerberg/axe-storybook-testing) | devDependencies | [`^6.1.0` -> `^6.2.0`](https://renovatebot.com/diffs/npm/@chanzuckerberg%2faxe-storybook-testing/6.1.0/6.2.0) |
| [@chanzuckerberg/eslint-config-edu-js](https://togithub.com/chanzuckerberg/frontend-libs/blob/main/packages/eslint-config-edu-js) ([source](https://togithub.com/chanzuckerberg/frontend-libs)) | devDependencies | [`^1.0.1` -> `^1.0.2`](https://renovatebot.com/diffs/npm/@chanzuckerberg%2feslint-config-edu-js/1.0.1/1.0.2) |
| [@chanzuckerberg/eslint-config-edu-ts](https://togithub.com/chanzuckerberg/frontend-libs/blob/main/packages/eslint-config-edu-ts) ([source](https://togithub.com/chanzuckerberg/frontend-libs)) | devDependencies | [`^1.0.0` -> `^1.0.4`](https://renovatebot.com/diffs/npm/@chanzuckerberg%2feslint-config-edu-ts/1.0.0/1.0.4) |
| [@chanzuckerberg/eslint-plugin-edu-react](https://togithub.com/chanzuckerberg/frontend-libs/blob/main/packages/eslint-plugin-edu-react) ([source](https://togithub.com/chanzuckerberg/frontend-libs)) | devDependencies | [`^1.1.0` -> `^1.1.7`](https://renovatebot.com/diffs/npm/@chanzuckerberg%2feslint-plugin-edu-react/1.1.0/1.1.7) |
| [@chanzuckerberg/story-utils](https://togithub.com/chanzuckerberg/frontend-libs/blob/main/packages/story-utils) ([source](https://togithub.com/chanzuckerberg/frontend-libs)) | devDependencies | [`^3.0.8` -> `^3.0.10`](https://renovatebot.com/diffs/npm/@chanzuckerberg%2fstory-utils/3.0.8/3.0.10) |
| [@commitlint/cli](https://commitlint.js.org/) ([source](https://togithub.com/conventional-changelog/commitlint)) | devDependencies | [`^16.2.1` -> `^16.3.0`](https://renovatebot.com/diffs/npm/@commitlint%2fcli/16.2.4/16.3.0) |
| [@commitlint/config-conventional](https://commitlint.js.org/) ([source](https://togithub.com/conventional-changelog/commitlint)) | devDependencies | [`^16.2.1` -> `^16.2.4`](https://renovatebot.com/diffs/npm/@commitlint%2fconfig-conventional/16.2.4/16.2.4) |
| [@commitlint/prompt-cli](https://commitlint.js.org/) ([source](https://togithub.com/conventional-changelog/commitlint)) | devDependencies | [`^16.2.1` -> `^16.3.0`](https://renovatebot.com/diffs/npm/@commitlint%2fprompt-cli/16.2.4/16.3.0) |
| [@headlessui/react](https://togithub.com/tailwindlabs/headlessui) | dependencies | [`^1.6.6` -> `^1.7.5`](https://renovatebot.com/diffs/npm/@headlessui%2freact/1.6.6/1.7.5) |
| [@storybook/addon-a11y](https://togithub.com/storybookjs/storybook/tree/main/addons/a11y) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2faddon-a11y/6.5.12/6.5.14) |
| [@storybook/addon-docs](https://togithub.com/storybookjs/storybook/tree/main/addons/docs) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2faddon-docs/6.5.12/6.5.14) |
| [@storybook/addon-essentials](https://togithub.com/storybookjs/storybook/tree/main/addons/essentials) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2faddon-essentials/6.5.12/6.5.14) |
| [@storybook/addon-interactions](https://togithub.com/storybookjs/storybook/tree/main/addons/interactions) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2faddon-interactions/6.5.12/6.5.14) |
| [@storybook/addon-links](https://togithub.com/storybookjs/storybook/tree/main/addons/links) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2faddon-links/6.5.12/6.5.14) |
| [@storybook/react](https://togithub.com/storybookjs/storybook/tree/main/app/react) ([source](https://togithub.com/storybookjs/storybook)) | devDependencies | [`^6.5.12` -> `^6.5.14`](https://renovatebot.com/diffs/npm/@storybook%2freact/6.5.12/6.5.14) |
| [@storybook/storybook-deployer](https://togithub.com/storybooks/storybook-deployer) | devDependencies | [`^2.8.12` -> `^2.8.16`](https://renovatebot.com/diffs/npm/@storybook%2fstorybook-deployer/2.8.12/2.8.16) |
| [@testing-library/react](https://togithub.com/testing-library/react-testing-library) | devDependencies | [`^12.1.4` -> `^12.1.5`](https://renovatebot.com/diffs/npm/@testing-library%2freact/12.1.5/12.1.5) |
| [@types/jest](https://togithub.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/jest) ([source](https://togithub.com/DefinitelyTyped/DefinitelyTyped)) | devDependencies | [`^29.0.3` -> `^29.2.4`](https://renovatebot.com/diffs/npm/@types%2fjest/29.0.3/29.2.4) |
| [@types/react](https://togithub.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react) ([source](https://togithub.com/DefinitelyTyped/DefinitelyTyped)) | devDependencies | [`^17.0.40` -> `^17.0.52`](https://renovatebot.com/diffs/npm/@types%2freact/17.0.45/17.0.52) |
| [@types/react-beautiful-dnd](https://togithub.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-beautiful-dnd) ([source](https://togithub.com/DefinitelyTyped/DefinitelyTyped)) | devDependencies | [`^13.1.2` -> `^13.1.3`](https://renovatebot.com/diffs/npm/@types%2freact-beautiful-dnd/13.1.2/13.1.3) |
| [@types/react-dom](https://togithub.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-dom) ([source](https://togithub.com/DefinitelyTyped/DefinitelyTyped)) | devDependencies | [`^17.0.13` -> `^17.0.18`](https://renovatebot.com/diffs/npm/@types%2freact-dom/17.0.17/17.0.18) |
| [chromatic](https://www.chromatic.com) ([source](https://togithub.com/chromaui/chromatic-cli)) | devDependencies | [`^6.9.0` -> `^6.11.4`](https://renovatebot.com/diffs/npm/chromatic/6.9.0/6.11.4) |
| [eslint](https://eslint.org) ([source](https://togithub.com/eslint/eslint)) | devDependencies | [`^8.23.1` -> `^8.29.0`](https://renovatebot.com/diffs/npm/eslint/8.23.1/8.29.0) |
| [eslint-plugin-jest](https://togithub.com/jest-community/eslint-plugin-jest) | devDependencies | [`^27.0.4` -> `^27.1.6`](https://renovatebot.com/diffs/npm/eslint-plugin-jest/27.0.4/27.1.6) |
| [eslint-plugin-testing-library](https://togithub.com/testing-library/eslint-plugin-testing-library) | devDependencies | [`^5.6.4` -> `^5.9.1`](https://renovatebot.com/diffs/npm/eslint-plugin-testing-library/5.6.4/5.9.1) |
| [flowgen](https://togithub.com/joarwilk/flowgen) | devDependencies | [`^1.14.1` -> `^1.20.1`](https://renovatebot.com/diffs/npm/flowgen/1.19.0/1.20.1) |
| [jest](https://jestjs.io/) ([source](https://togithub.com/facebook/jest)) | devDependencies | [`^29.0.3` -> `^29.3.1`](https://renovatebot.com/diffs/npm/jest/29.0.3/29.3.1) |
| [jest-environment-jsdom](https://togithub.com/facebook/jest) | devDependencies | [`^29.0.3` -> `^29.3.1`](https://renovatebot.com/diffs/npm/jest-environment-jsdom/29.0.3/29.3.1) |
| [lint-staged](https://togithub.com/okonet/lint-staged) | devDependencies | [`^13.0.3` -> `^13.1.0`](https://renovatebot.com/diffs/npm/lint-staged/13.0.3/13.1.0) |
| [plop](https://plopjs.com) ([source](https://togithub.com/plopjs/plop)) | devDependencies | [`^3.0.5` -> `^3.1.1`](https://renovatebot.com/diffs/npm/plop/3.1.0/3.1.1) |
| [postcss](https://postcss.org/) ([source](https://togithub.com/postcss/postcss)) | devDependencies | [`^8.4.12` -> `^8.4.19`](https://renovatebot.com/diffs/npm/postcss/8.4.13/8.4.19) |
| [postcss-import](https://togithub.com/postcss/postcss-import) | devDependencies | [`^14.0.2` -> `^14.1.0`](https://renovatebot.com/diffs/npm/postcss-import/14.1.0/14.1.0) |
| [postcss-mixins](https://togithub.com/postcss/postcss-mixins) | devDependencies | [`^9.0.2` -> `^9.0.4`](https://renovatebot.com/diffs/npm/postcss-mixins/9.0.2/9.0.4) |
| [react-focus-lock](https://togithub.com/theKashey/react-focus-lock) | dependencies | [`^2.9.1` -> `^2.9.2`](https://renovatebot.com/diffs/npm/react-focus-lock/2.9.1/2.9.2) |
| [style-dictionary](https://togithub.com/amzn/style-dictionary) | devDependencies | [`^3.7.0` -> `^3.7.1`](https://renovatebot.com/diffs/npm/style-dictionary/3.7.0/3.7.1) |
| [stylelint](https://stylelint.io) ([source](https://togithub.com/stylelint/stylelint)) | devDependencies | [`^14.12.0` -> `^14.16.0`](https://renovatebot.com/diffs/npm/stylelint/14.12.0/14.16.0) |
| [tailwindcss](https://tailwindcss.com) ([source](https://togithub.com/tailwindlabs/tailwindcss)) | devDependencies | [`^3.1.8` -> `^3.2.4`](https://renovatebot.com/diffs/npm/tailwindcss/3.1.8/3.2.4) |
| [ts-jest](https://kulshekhar.github.io/ts-jest) ([source](https://togithub.com/kulshekhar/ts-jest)) | devDependencies | [`^29.0.1` -> `^29.0.3`](https://renovatebot.com/diffs/npm/ts-jest/29.0.1/29.0.3) |

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

**Comment by jinlee93** [comment]
IT WORKS (mostly) 
- didn't attach reviewers tho 🧐

File: package.json [hunk 1]
```diff
@@ -95,62 +95,62 @@
...
   "devDependencies": {
-    "@chanzuckerberg/axe-storybook-testing": "^6.1.0",
-    "@chanzuckerberg/eslint-config-edu-js": "^1.0.1",
-    "@chanzuckerberg/eslint-config-edu-ts": "^1.0.0",
-    "@chanzuckerberg/eslint-plugin-edu-react": "^1.1.0",
+    "@chanzuckerberg/axe-storybook-testing": "^6.2.0",
+    "@chanzuckerberg/eslint-config-edu-js": "^1.0.2",
+    "@chanzuckerberg/eslint-config-edu-ts": "^1.0.4",
+    "@chanzuckerberg/eslint-plugin-edu-react": "^1.1.7",
     "@chanzuckerberg/eslint-plugin-stories": "^3.2.2",
```
Comment (jinlee93) [inline package.json:102]: Latest is 3.2.7 but the `no-components-without-story` rule flags components with a story file extension of `TS`, will wait until https://github.com/chanzuckerberg/frontend-libs/pull/41 and new publish

File: package.json [hunk 2]
```diff
@@ -95,62 +95,62 @@
...
     "@types/node": "^17.0.21",
-    "@types/react": "^17.0.40",
-    "@types/react-beautiful-dnd": "^13.1.2",
-    "@types/react-dom": "^17.0.13",
+    "@types/react": "^17.0.52",
+    "@types/react-beautiful-dnd": "^13.1.3",
+    "@types/react-dom": "^17.0.18",
     "@types/react-portal": "^4.0.4",
     "@types/svg4everybody": "^2.1.2",
     "axe-core": "^4.4.3",
```
Comment (jinlee93) [inline package.json:130]: There's a v4.5 but for some reason causes some stories to time out

File: package.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
     "build:tokens": "rm -rf src/tokens-dist/ && node ./style-dictionary.config.js && yarn prettier-tokens-dist && yarn copy-tokens-to-lib",
     "build:icons": "svg-sprite-generate -d src/icons/ -o src/icons/spritemap/spritemap.svg && yarn generate-icon-types",
     "build:js": "tsc --project tsconfig.build.json",
-    "build:storybook": "build-storybook -o storybook-static -s src/design-tokens/tier-1-definitions/fonts",
+    "build:storybook": "NODE_OPTIONS=--openssl-legacy-provider build-storybook -o storybook-static -s src/design-tokens/tier-1-definitions/fonts",
```
Comment (jinlee93) [inline package.json:29]: This snuck in here, but is required for running locally

File: src/components/Select/Select.tsx [hunk 4]
```diff
@@ -166,6 +166,7 @@ export function Select(props: SelectProps) {
     className,
     children,
     'aria-label': ariaLabel,
+    value = null,
```
Comment (jinlee93) [inline src/components/Select/Select.tsx:169]: With HeadlessUI update, this complains if it's empty that it's changing from uncontrolled to controlled

File: src/components/Dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -258,6 +258,7 @@ export function Dropdown(props: DropdownProps) {
     options,
     children,
     'aria-label': ariaLabel,
+    value = null,
```
Comment (jinlee93) [inline src/components/Dropdown/Dropdown.tsx:261]: With HeadlessUI update, this complains if it's empty that it's changing from uncontrolled to controlled

Comment (dierat) [reply src/components/Dropdown/Dropdown.tsx:261]: Do you think this is worth a code comment? Maybe just something about it being required by Headless UI.

**Review by dierat — APPROVED** [review]
Thanks for handling this!

**Review by booc0mtaco — APPROVED** [review]
That's a lot of snapshot changes for a minor release!

File: src/components/Dropdown/Dropdown.tsx (hunk 5 above)
Comment (jinlee93) [reply src/components/Dropdown/Dropdown.tsx:261]: Makes sense, will add comment for both files

**Pull request merged**